components/MobileNavigation.js

```diff
@@ -37,6 +47,26 @@ export default class MobileNavigation extends Component {
     } else {
       if (this.state.showMobileNav) this.hideMobileNav()
     }
+    const result = this.findScrollSection()
+    this.setState({
```

Comment:
If the result doesn’t change, there is no need to update the state (which triggers a rerender).

Better guard it:

```js
    const result = this.findScrollSection()
    if (this.state.activeMenu !== result) {
      this.setState({ nowSelectMenu: result })
    }
```